From 50f2ada6cbfafa75b628410e8834f29581854e6f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Thalheim?= <joerg@thalheim.io>
Date: Sun, 6 Dec 2020 08:34:19 +0100
Subject: [PATCH 18/19] pkg-config: derive prefix from --prefix

Point prefix to the one configured, instead of `/usr` `systemd` has limited
support for making the pkgconfig prefix overridable, and interpolates those
values later down.

So we only need to patch this one value to get the correct paths.
See systemd/systemd@bc4e6e27922a2873985ab9367d79fb099f70b505 for details.

Co-Authored-By: Florian Klink <flokli@flokli.de>
---
 src/core/systemd.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/systemd.pc.in b/src/core/systemd.pc.in
index 162432e77f..2fc20daf03 100644
--- a/src/core/systemd.pc.in
+++ b/src/core/systemd.pc.in
@@ -11,7 +11,7 @@
 # considered deprecated (though there is no plan to remove them). New names
 # shall have underscores.
 
-prefix=/usr
+prefix={{PREFIX}}
 root_prefix={{ROOTPREFIX_NOSLASH}}
 rootprefix=${root_prefix}
 sysconf_dir={{SYSCONF_DIR}}
-- 
2.33.1

